tests/application_development/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+FILE(GLOB app_sources src/*.cpp)
```

Comment:
I think @SebastianBoe prefers not using GLOB in Zephyr's CMake files.

Reply:
git grep "GLOB app_sources" | wc -l
261

Reply:
Cool, never mind :+1: